.travis.yml

```diff
@@ -70,6 +72,7 @@ matrix:
             - python-docutils
             - pngcrush
             - python-sphinx
+	    - python3-sphinx
```

Comment:
While this will probably work, the whitespace should be consistent.

Reply:
fixed in a fixup commit.